ui.apps/src/main/content/jcr_root/apps/asset-share-commons/clientlibs/clientlib-site/js/stores/profile-store.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * Asset Share Commons
+ *
+ * Copyright [2020]  Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Utility specific to storing data about the users Profile
+ */
+AssetShare.Store.Profile = (function (ns, store) {
+    'use strict';
+
+    var PROFILE_KEY       = 'profile',
+        currentUserId;
+
+    /**
+     * currentUserId is set
+     */
+    function isReady() {
+        return currentUserId !== undefined && currentUserId.length > 0;
```

Comment:
```suggestion
        return currentUserId && currentUserId.length > 0;
```

Or could be `typeof currentUserId !== 'undefined'` for explicity undefined check (vs. just a truthy check)